scripts/get-translator-credits.py

```diff
@@ -60,7 +59,7 @@ def get_language_name(locale_string):
 language_names.sort()
 
 for (language_name, locale) in language_names:
-    print(("%s - %s" % (language_name, locale)))  # noqa: T201
+    print("{} - {}".format(language_name, locale))  # noqa: T201
```

Comment:
Curious: this changes `%`-formatting to `.format()`, but in other places we have `.format()` being replaced with `f`-strings. Does this mean re-running pyupgrade will change this to use f-string?

Reply:
@laymonage ha! didn't occur to me to try again. It does. Shall I commit and push?

Reply:
Would be nice to be consistent, but I don't know if @gasman has started reviewing or not

Reply:
pushed

Reply:
I had, but I heartily approve of this update - having to context-switch between the different formatting variants was making the reviewing slower than it needed to be :-)